integration_tests/tests/apply/risks-and-needs/risk-information/complete_acct_page.cy.ts

```diff
@@ -109,12 +107,36 @@ context('Visit "Concerns and health needs" section', () => {
     page.hasNoAccts()
   })
 
+  //  Scenario: change my answer when no ACCTs exist
+  //    Given no ACCTs exist
+  //    And I am on the ACCT page
+  //    When I click "Change your answer about existing ACCT notes"
+  //    Then I am taken to the "Does the applicant have ACCT notes" page
+  it('navigates to the "Does the applicant have ACCT notes" page', function test() {
+    // cy.task('stubApplicationGet', { application: this.application })
+    // AcctPage.visit(this.application)
+
+    const page = new AcctPage(this.application)
+
+    //    Given no ACCTs exist
+    page.hasNoAccts()
+
+    //    When I click "Change your answer about existing ACCT notes"
+    page.clickLink('Change your answer about existing ACCT notes')
+
+    //    Then I can navigate to the "Does the applicant have ACCT notes" page
+    Page.verifyOnPage(ApplicantAcctNotesPage, this.application)
+  })
+
   //  Scenario: complete page and navigate to next page in health needs task
   //    When I continue to the next task / page
   //    Then I see the "violence and arson" page
   it('navigates to the next page (violence and arson)', function test() {
-    AcctPage.visit(this.application)
-    const page = new AcctPage(this.application)
+    // When there is already imported data
```

Comment:
what does 'imported data' mean here?

Reply:
Whoops, this was copied from another file I think.